specification/security/resource-manager/Microsoft.Security/preview/2020-02-01-preview/iotSecuritySolutions.json

```diff
@@ -0,0 +1,734 @@
+              },
+              {
+                "value": "IoT_PermissiveFirewallPolicy",
+                "description": "An Allowed firewall policy was found (INPUT/OUTPUT). The policy should Deny all traffic by default and define rules to allow necessary communication to/from the device."
+              },
+              {
+                "value": "IoT_PermissiveInputFirewallRules",
+                "description": "A rule in the firewall has been found that contains a permissive pattern for a wide range of IP addresses or Ports."
+              },
+              {
+                "value": "IoT_PermissiveOutputFirewallRules",
+                "description": "A rule in the firewall has been found that contains a permissive pattern for a wide range of IP addresses or Ports."
+              },
+              {
+                "value": "IoT_PrivilegedDockerOptions",
+                "description": "Edge module is configured to run in privileged mode, with extensive Linux capabilities or with host-level network access (send/receive data to host machine)."
+              },
+              {
+                "value": "IoT_SharedCredentials",
+                "description": "Same authentication credentials to the IoT Hub used by multiple devices. This could indicate an illegitimate device impersonating a legitimate device. It also exposes the risk of device impersonation by an attacker."
+              },
+              {
+                "value": "IoT_VulnerableTLSCipherSuite",
+                "description": "Insecure TLS configurations detected. Immediate upgrade recommended."
+              }
+            ]
+          }
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true
+        },
+        "status": {
+          "type": "string",
+          "enum": [
+            "Disabled",
+            "Enabled"
+          ],
+          "default": "Enabled",
+          "description": "Recommendation status. When the recommendation status is disabled recommendations are not generated.",
+          "x-ms-enum": {
+            "name": "RecommendationConfigStatus",
+            "modelAsString": true
+          }
+        }
+      },
+      "required": [
+        "recommendationType",
+        "status"
+      ]
+    },
+    "RecommendationConfigurationList": {
+      "type": "array",
+      "description": "List of the configuration status for each recommendation type.",
+      "items": {
+        "$ref": "#/definitions/RecommendationConfigurationProperties"
+      }
+    },
+    "AdditionalWorkspaceProperties": {
+    "type": "object",
+    "description": "The properties of an additional workspace",
+    "properties": {
+      "workspace": {
+        "type": "string",
+        "description": "The ARM ID of the workspace"
+      },
+      "type": {
+        "type": "string",
+        "description": "The type of the workspace",
+        "enum": [
+          "Sentinel"
+        ]
```

Comment:
Only the following two fields don't include `x-ms-enum`. Is it on purpose?

Reply:
Opened a new PR with the requested changes
https://github.com/Azure/azure-rest-api-specs/pull/8545